libraries/AP_Logger/AP_Logger_Backend.h

```diff
@@ -162,6 +162,14 @@ class AP_Logger_Backend
 
     virtual void io_timer(void) {}
 
+    enum class Backend_Type : uint8_t {
+      NONE       = 0,
```

Comment:
We should eliminate this entry.  Now if possible, on a future PR if there's even a single user.